dispatcher-consumer/handler.go

```diff
@@ -1,15 +1,88 @@
 package dispatcherconsumer
 
 import (
+	"config-manager/domain/message"
+	kafkaUtils "config-manager/infrastructure/kafka"
 	"context"
+	"encoding/json"
 	"fmt"
+	"log"
+
+	"github.com/google/uuid"
 
 	kafka "github.com/segmentio/kafka-go"
 )
 
 type handler struct {
+	producer      kafkaUtils.KafkaWriterInterface
+	uuidGenerator func() uuid.UUID
+}
+
+// This message to inventory is constructed using data from provided labels. This will change
+// once this information can be consumed from the run_hosts topic.
+func buildMessage(payload message.DispatcherEventPayload, reqID uuid.UUID) ([]byte, error) {
+	msg := message.InventoryUpdate{
+		Operation: "add_host",
+		Metadata:  message.PlatformMetadata{RequestID: reqID.String()},
```

Comment:
I think the inventory service is going to start requiring the base64 encoded identity header to get passed along as part of the platform_metadata.  I'm not sure how you are going to get an identity header value here though...with this update getting triggered asynchronously via playbook-dispatcher.

We'll need to confirm that requirement.

Let me know if we need to discuss.

I think you'll also be required to pass along a "reporter" field.

Reply:
I was told a reporter wouldn't be necessary for updating via this topic, but I'll double-check to confirm. As for the identity header.. yeah not sure how I'd get that currently.

Reply:
Oh!  I missed that this was getting sent to a new topic. My bad!

Reply:
Np! This is the new system-profile topic for non-reporters